dracut/30ignition/persist-ifcfg.sh

```diff
@@ -33,6 +35,32 @@ persist_ifcfg() {
         return 0
     fi
 
+    # Persist the hostname if the admin has elected to use the dracut method of
+    # defining the IP on the commandline.
+    hpath="/sysroot/etc/hostname"
+    hname=$(< /proc/sys/kernel/hostname)
+    for iface in $(ls /sys/class/net)
```

Comment:
Need to skip this loop if `hpath` exists already

Reply:
Fixed up.